tests/utils.py

```diff
@@ -391,33 +383,58 @@ def cleanse(self, py_input, substitutions):
         return out
 
 
-def _try_eval(value):
-    if value and value.startswith('||| '):
-        value = value[4:]
-        try:
-            value = eval(value)
-        except:
-            pass
-    return value
+def _normalize(value):
+    """
+    ||| -- lines starting with this pattern will be `eval`uated and compared as
+        native python objects. Mostly used for output data from scripts.
+    /// -- error messages might print out information from the object that
+        generated it. This might lead to failure due to the ordering randomness
+        of some object types. Lines starting with this pattern will be treated specially
+        as to overcome this problem.
+
+    Notice that a line might start with '||| ///'. This is helpful for string replacement cases.
+    """
+    native = value
+    if value:
+        if value.startswith('||| '):
+            value = value[4:]
+            try:
+                native = eval(value)
+            except:
+                pass
 
+        if value.startswith('/// '):
+            value = value[4:]
+            native = collections.Counter(value)
```

Comment:
@freakboy3742 this is a heuristic. IMHO It looks reasonable for our use case but please review carefully as it can lead to false positives.